samples/sampler/stories/default/IncrementSlider.js

```diff
@@ -33,6 +33,7 @@ storiesOf('Moonstone', module)
 					onChange={action('onChange')}
 					orientation={select('orientation', ['horizontal', 'vertical'], IncrementSliderConfig)}
 					step={number('step', IncrementSliderConfig)} // def: 1
+					style={{marginLeft: '96px',	marginRight: '96px'}}
```

Comment:
why did you use tab?
```suggestion
					style={{marginLeft: '96px', marginRight: '96px'}}
```

Reply:
I think, it is not a matter of margin, but the direction of the tooltip is reversed in RTL and LTL . Did you see the relevant part?

Reply:
one more thing. Slider sample has same issue. Please consider it at once

Reply:
The tooltip should appear in relation to the component by following rules for the `'Side'` prop.
`'left'` - Left of the component
`'before'` - Start of text side of the component
`'right'` - Right of the component
`'after'` - End of text side of the component
So when `orientation = 'vertical'` and `side = 'before'`, it should actually appear left of the component in LTR and right of the component in RTL.
I think the sample works fine except for displaying issue.

Reply:
I just found Silder sample has the same issue. Thank you for notifying me.

Reply:
insted of "px, please use ri.scaleToRem

Reply:
Slider and ProgressBar sample has the same issue, fixed it.
And changed to use ri.scaleToRem.